relational_operators/CreateIndexOperator.hpp

```diff
@@ -75,6 +84,8 @@ class CreateIndexOperator : public RelationalOperator {
  private:
   CatalogRelation *relation_;
   const std::string &index_name_;
+  const std::vector<IndexSubBlockDescription> index_descriptions_;
```

Comment:
I tried using a reference here. However, sometimes the index_descriptions_ vector object, when invoked in a different thread from CreateIndexOperator, is cleared out. I am investigating this and meanwhile have left this unchanged.